tests/urls.py

```diff
@@ -2,6 +2,6 @@
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```

Comment:
Should we remove the admin pattern? If we keep it, I think we need to add a `DATABASES` setting, so the admin url won't error if someone is running this site for development purposes.

Reply:
Think we should just get rid of it. This app has no models, no sign-in, or anything that would warrant an admin site.